platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java

```diff
@@ -102,6 +104,24 @@ default void preHandle(final Event event) {}
      */
     void handleConsensusRound(final Round round, final SwirldDualState swirldDualState);
 
+    /**
+     * Implementations of the SwirldState should always override this method in production.  The AddressBook returned
```

Comment:
as I understand it now, the address book being passed here might not necessarily be the config address book, but it depends. best to just call it address book and not mention where it comes from in the comments are variable name